tests/visual/bottom-nav.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Bottom Navigation</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            grid-template-columns: repeat(1, 1fr);
+            background-color: #F3F5F7;
+        }
+    </style>
+</head>
+<body>
+
+    <div id="test-area" class="grid">
+        <section>
+            <h4>Horizontal</h4>
+            <nav class="k-bottom-nav k-bottom-nav-primary k-bottom-nav-flat k-bottom-nav-border k-bottom-nav-item-flow-horizontal">
```

Comment:
Do we have a design update here as now I see that we have to expose `fill` prop in the API of the component that accepts `solid` and `flat` values. Also, we need to add it in the spec.

Reply:
The fill prop allows to differentiate between solid and flat theme variations and the change is synced with @tyanko . Design is not updated yet with latest changes.

Already added in the spec #1949